vendor/README.md

```diff
@@ -202,6 +202,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Git Config:** [Alhadis/language-etc](https://github.com/Alhadis/language-etc)
 - **Git Revision List:** [Nixinova/NovaGrammars](https://github.com/Nixinova/NovaGrammars)
 - **Gleam:** [gleam-lang/tree-sitter-gleam](https://github.com/gleam-lang/tree-sitter-gleam) 🐌
+- **Glimmer:** [atom/language-javascript](https://github.com/atom/language-javascript)
```

Comment:
Not sure if this should link to https://github.com/lifeart/vsc-ember-syntax instead. This output was generated by running `script/add-grammar https://github.com/lifeart/vsc-ember-syntax`

Reply:
Is it worth fixing?

Reply:
Yes, I'll set it manually given the purpose of this file